lib/temporal/testing/scheduled_workflows.rb

```diff
@@ -0,0 +1,55 @@
+module Temporal
+  module Testing
+    class WorkflowIDNotScheduled < ClientError; end
+
+    # Implementation for Temporal::Testing::ScheduledWorkflwows
```

Comment:
This is not really providing implementation for `ScheduledWorkflow` does it? As it there's nothing connecting the two (even the interface is different), so maybe call it `ScheduledWorkflowStore` or something?